apps/meteor/client/views/root/AppErrorPage.tsx

```diff
@@ -15,6 +15,17 @@ const AppErrorPage = (_props: AppErrorPageProps): ReactElement => {
 				<StatesIcon name='error-circle' />
 				<StatesTitle>Application Error</StatesTitle>
 				<StatesSubtitle>The application GUI just crashed.</StatesSubtitle>
```

Comment:
`GUI` for end users may be too technical

I suggest a more friendly text to make the user more comfortable in use the new feature:

Oops! Something Went Wrong
We encountered an unexpected error and need to reload the application.

What Happened:
The application experienced an issue that requires a reload to continue.

What to Do:
Click the button below to reload the application.
If the problem persists, please contact our support team.
[Reload Application]

We apologize for the inconvenience and appreciate your understanding.

Reply:
I understand and agree with your suggestion, but this is a UI improvement, there are several improvements (actually showing the error that occurred, etc.) and it is not the objective of the pr here, I am 100% in favor of improving this page, just not in this pr